src/mycanvas.h

```diff
@@ -114,7 +115,8 @@ struct TSCanvas : public wxScrolledWindow {
         // (scrolling) don't work.
         // The 128 makes sure unicode entry on e.g. Polish keyboards still works.
         // (on Linux in particular).
-        if ((ce.GetModifiers() == wxMOD_ALT) && (ce.GetUnicodeKey() < 128)) {
+        // Special case: ALT+RETURN shall be processed.
+        if (ce.GetKeyCode() != WXK_RETURN && (ce.GetModifiers() == wxMOD_ALT) && (ce.GetUnicodeKey() < 128)) {

```

Comment:
`enter` or `alt + enter` is used in the menus?
For consistency, remove () from the latter 2 conditions, or add it to the first

Reply:
I revisited the wxWidgets event handling and got to know, that shortcuts are auto-assigned by MyAppend. I simplified my implementation.